bin/etest

```diff
@@ -56,6 +57,10 @@ $(opt_parse \
     "+verbose v=${VERBOSE:-0}  | Verbose output." \
     ":work_dir                 | Temporary location where etest can place temporary files.  This location will be both
                                  created and deleted by etest." \
+    "+coverage                 | Run etest through kcov coverage tool to identify and report code coverage for all
+                                 executed code." \
```

Comment:
This is going to be amazing!

Reply:
yes, very much agree. Early protoypes of this look amazeballs. I put this on hold as I had more urgent things I had to finish, but I'm going to come back to this soon. BTW, speaking of **AMAZING** check this out:

https://elibs.github.io/ebash